toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -1340,7 +1340,7 @@ func addUsers(installChroot *safechroot.Chroot, users []configuration.User) (err
 			return
 		}
 
-		err = ProvisionUserSSHCerts(installChroot, user.Name, user.SSHPubKeyPaths, user.SSHPubKeys)
+		err = ProvisionUserSSHCerts(installChroot, user.Name, user.SSHPubKeyPaths, user.SSHPubKeys, false)
```

Comment:
Could you either use a name constant or the `/* ... */` comment so it's clearer what `false` means?

Reply:
Fixed.